components/renku_data_services/notebooks/api.spec.yaml

```diff
@@ -723,6 +876,158 @@ components:
       - renku.io/projectName
       - renku.io/repository
       type: object
+    SessionPostRequest:
```

Comment:
Allow sending cloud storage details `cloudstorage ` , as it's useful when storage credentials are missing.

Reply:
Done.